config/add-on-manager/manager.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: kuadrant-add-on-manager
-  namespace: system
+  namespace: multicluster-gateway-controller-system
```

Comment:
This should always be getting called from an overlay with `namespace: foo` set so you don't need to hard code the namespace here